img/laminas-components-rgb.svg

```diff
@@ -1 +1,98 @@
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This changes brings back the old and wrong aligned logo.

Reply:
this is what gulp call did ... I think we haven't updated them in assets directory?

Reply:
Uff, maybe.

Reply:
Right: https://github.com/laminas/laminas.github.io/commit/d470e72fe863f54dd3cf1c83f5496c01881f7876

Reply:
Reverted and updated in asset/img now